ReactTask/src/modules/Redux.modules.ts

```diff
@@ -0,0 +1,74 @@
+export interface AuthReducerType {
+  user?: {
+    id: string;
+    first_name: string;
+    last_name: string;
+    photo: string;
+    role_name: string;
+  };
+  token?: string;
+  error?: string;
+  isAuthenticated: boolean;
+}
+
+export interface IAddAppointmentResponse {
+  id: string;
+  patient_id: string;
+  doctor_id: string;
+  visit_date: string;
+  reason: string;
+  note: string;
+  status: string;
+}
+
+export interface IAddAppointment {
+  date: string;
+  reason: string;
+  note: string;
+  doctorID: string;
+}
+
+export interface IAddAppointmentResponse {
+  id: string;
+  patient_id: string;
+  doctor_id: string;
+  visit_date: string;
+  reason: string;
+  note: string;
+  status: string;
+}
+
+export interface ILogInForm {
+  userName: string;
+  password: string;
+}
+
+export interface IRegisterForm {
+  userName: string;
+  password: string;
+  firstName: string;
```

Comment:
You could extend these interfaces
IRegisterForm & ILogInForm

Reply:
Fixed